llvm/include/llvm/Transforms/udachinaPass/udachinaPass.h

```diff
@@ -16,4 +16,4 @@ namespace llvm {
 	};
 
 }
-#endif
+
```

Comment:
@anastasiaudachina, why you've moved this to cpp file?

Reply:
@geexie it is from lab2. You asked me to move this part to the cpp file:
#ifndef LLVM_TRANSFORMS_UDACHINAPASS_H
#define LLVM_TRANSFORMS_UDACHINAPASS_H
#define DEBUG_TYPE "udachinaPass"

So I added #endif to follow the construction

Reply:
I meant #define just `DEBUG_TYPE "udachinaPass"`  and keep 

```
#ifndef LLVM_TRANSFORMS_UDACHINAPASS_H
#define LLVM_TRANSFORMS_UDACHINAPASS_H
```
 in header as a [header guard](https://www.learncpp.com/cpp-tutorial/header-guards/)

Reply:
@geexie I'm sorry, I misunderstood at first. I fixed it